src/main/java/com/google/codeu/servlets/AboutMeServlet.java

```diff
@@ -0,0 +1,22 @@
+package com.google.codeu.servlets;
+
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/*
+ * Temp-hard coded msg to test whether things show up under /about
+ */
+
+@WebServlet("/about")
+public class AboutMeServlet extends HttpServlet {
+
+	@Override
```

Comment:
please use 2 spaces instead of a tab; the rest of the code uses spaces so it'd be neater to be consistent.